assets/src/dashboard/components/popoverMenu/popoverCard.js

```diff
@@ -68,27 +57,23 @@ const MenuCounterRevealer = styled.div`
   border-radius: 8px;
 `;
 
-const ButtonInner = styled.div`
+const ButtonInner = styled(Popover)`
   ${fullSize}
-  position: absolute;
-  z-index: ${Z_INDEX.POPOVER_MENU};
-  opacity: ${({ isOpen }) => (isOpen ? 1 : 0)};
-  pointer-events: ${({ isOpen }) => (isOpen ? 'auto' : 'none')};
   transform: ${(props) => {
     switch (props.align) {
-      case CORNER_DIRECTIONS.TOP_LEFT:
+      case CORNER_DIRECTIONS.top_left:
```

Comment:
Took me a little bit to realize why these had to be lowercase, but, ha, I get it! 👍

Reply:
lol same